net-dns/host/host-20070128-r1.ebuild

```diff
@@ -1,8 +1,8 @@
-# Copyright 1999-2013 Gentoo Foundation
+# Copyright 1999-2015 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI="2"
+EAPI=5
```

Comment:
This ebuild has stable keywords. For EAPI bump, you must revbump it and do the changs on new revision.